Anlab.Mvc/Startup.cs

```diff
@@ -67,7 +67,7 @@ public void ConfigureServices(IServiceCollection services)
             // Add application services.
             services.AddTransient<IEmailSender, AuthMessageSender>();
             services.AddTransient<ISmsSender, AuthMessageSender>();
-            services.AddTransient<IDirectorySearchService, DirectorySearchService>();
+            services.AddTransient<IDirectorySearchService, DirectorySearchService>();

```

Comment:
@jSylvestre not sure why git thinks this is different but this is where the conflict happens